### ISSUE TITLE
Simulator runtimes format breakage

### ISSUE THREAD

**Issue by fbecart** [openbakery/gradle-xcodePlugin#368]
Instead of the following format:
```
== Runtimes ==
iOS 10.0 (10.0 - 14A5261u) (com.apple.CoreSimulator.SimRuntime.iOS-10-0)
tvOS 10.0 (10.0 - 14T5253q) (com.apple.CoreSimulator.SimRuntime.tvOS-10-0)
watchOS 3.0 (3.0 - 14S5247q) (com.apple.CoreSimulator.SimRuntime.watchOS-3-0)
```
`/Applications/Xcode.app/Contents/Developer/usr/bin/simctl list` returns:
```
== Runtimes ==
iOS 11.0 (11.0 - 15A372) - com.apple.CoreSimulator.SimRuntime.iOS-11-0
tvOS 11.0 (11.0 - 15J380) - com.apple.CoreSimulator.SimRuntime.tvOS-11-0
watchOS 4.0 (4.0 - 15R372) - com.apple.CoreSimulator.SimRuntime.watchOS-4-0
```

Notice how the identifier is separated from the rest by a dash instead of parentheses. I think the issue happens on my computer since the upgrade to XCode 9.0.

**Comment by igorkotkovets** [comment]
This change has broken the build environment on my project. :(. Are there any hot fixes planning?

`17:51:12.031 [INFO] [org.openbakery.CommandRunner]     Watch: Apple Watch Series 2 - 42mm (34246987-0CA5-4ECD-AC9A-AEF1DB1D1AFA) (Shutdown)
17:51:12.031 [INFO] [org.openbakery.CommandRunner]     Phone: iPhone 7 Plus (E492246A-5A3B-439E-87C6-58B2EC506273) (Shutdown)
17:51:12.099 [DEBUG] [org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter] Finished executing task ':xcodetest'
17:51:12.099 [LIFECYCLE] [class org.gradle.TaskExecutionLogger] :xcodetest FAILED
17:51:12.099 [INFO] [org.gradle.execution.taskgraph.AbstractTaskPlanExecutor] :xcodetest (Thread[main,5,main]) completed. Took 0.33 secs.
17:51:12.099 [DEBUG] [org.gradle.execution.taskgraph.AbstractTaskPlanExecutor] Task worker [Thread[main,5,main]] finished, busy: 1.185 secs, idle: 0.001 secs
17:51:12.110 [ERROR] [org.gradle.BuildExceptionReporter] 
17:51:12.110 [ERROR] [org.gradle.BuildExceptionReporter] FAILURE: Build failed with an exception.
17:51:12.110 [ERROR] [org.gradle.BuildExceptionReporter] 
17:51:12.110 [ERROR] [org.gradle.BuildExceptionReporter] * What went wrong:
17:51:12.110 [ERROR] [org.gradle.BuildExceptionReporter] Execution failed for task ':xcodetest'.
17:51:12.111 [ERROR] [org.gradle.BuildExceptionReporter] > Cannot invoke method hashCode() on null object
17:51:12.111 [ERROR] [org.gradle.BuildExceptionReporter] 
17:51:12.111 [ERROR] [org.gradle.BuildExceptionReporter] * Try:
17:51:12.111 [ERROR] [org.gradle.BuildExceptionReporter] Run with --stacktrace option to get the stack trace. 
17:51:12.111 [LIFECYCLE] [org.gradle.BuildResultLogger] 
17:51:12.111 [LIFECYCLE] [org.gradle.BuildResultLogger] BUILD FAILED
17:51:12.111 [LIFECYCLE] [org.gradle.BuildResultLogger] 
17:51:12.111 [LIFECYCLE] [org.gradle.BuildResultLogger] Total time: 16.645 secs`

**Comment by fbecart** [comment]
It's actually been fixed already in `develop` (see this commit https://github.com/openbakery/gradle-xcodePlugin/commit/22f7a5acfda608a5680793961826bf640de4c3a9#diff-1525aab4870d234adecdb4d09a038d92).

**Comment by igorkotkovets** [comment]
@fbecart , thank you for fast answer. Switching to `develop` version of plugin solved the issue.
I used `classpath "org.openbakery:xcode-plugin:0.15.1.develop.+"`

**Comment by phatblat** [comment]
I believe this can be closed now that version 0.15.1 has been released.
